src/mlagility/analysis/analysis.py

```diff
@@ -137,6 +137,8 @@ def call_benchit(
             model_info.status_message = "Model successfully built!"
         model_info.status_message_color = printing.Colors.OKGREEN
 
+        status.update(tracer_args.models_found)
```

Comment:
The code is still setting the `model_info.status_message` in the `except:` blocks even though the `status.update(tracer_args.models_found)` is in the `try` block. 

I guess that means that if you have two models, and the first one hits an exception, then the exception status will be printed for the first model the `status.update(tracer_args.models_found)` is called for the second model. But if there is only one model, and it raises an exception, then we never see the status for that model?

Suggestions:
1. Eliminate the inconsistent behavior, exceptions should look the same for both single- and multi-model runs
2. Eliminate dead code, such as `status_message = ` if it should never be encountered/used

Reply:
Also please paste the old and new behavior into the PR description so that I don't have to repro myself 😉

Reply:
please have a second look now